scripts/scil_evaluate_bundle_saturation.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This script performs a bundle saturation analysis (SATA) on a given tractogram,
+either as a standalone bundle or within the context of a whole-brain tractogram.
+
+It simulates a sampling of your bundle/tractogram and shows how the metrics
+evolve with the number of streamlines picked. This allows you to determine
+the number of streamlines needed to get a stable values.
+
+Input:
+- A tractogram file representing a bundle of interest (`bundle` argument).
+- Optionally, a whole-brain tractogram for comprehensive analysis
+    (`--whole_brain` argument).
+Both of these (if used) should be extremely dense tractograms, to ensure
+more than enough streamlines are available for sampling.
+
+Output:
+- A JSON file containing metrics such as volume, dice coefficient, entropy,
+    etc., computed at different sampling levels.
+- A set of PNG plots for each metric, illustrating how the metric values
+    change with different sampling levels.
```

Comment:
Should we explain these metric/plots?
I just ran the SATA script on a single bundle and I have to admit that I'm a bit lost in the different PNGs produced.